app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    hidden = False
```

Comment:
The `hidden` attribute is defined as a class variable, but it should be an instance variable. Each animal instance should have its own `hidden` state, so it should be defined within the `__init__` method.